src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Core/Utils/Mappers/DtoMapperConnectionQuery.cs

```diff
@@ -0,0 +1,105 @@
+using Altinn.AccessMgmt.PersistenceEF.Models;
+using Altinn.AccessMgmt.PersistenceEF.Queries.Connection.Models;
+using Altinn.Authorization.Api.Contracts.AccessManagement;
+
+namespace Altinn.AccessMgmt.Core.Utils;
+
+/// <inheritdoc/>
+public partial class DtoMapper : IDtoMapper
+{
+    public static List<ConnectionDto> Convert(IEnumerable<ConnectionQueryExtendedRecord> res)
+    {
+        var connections = res.GroupBy(res => res.To.Id);
+        var result = connections.Select(c =>
+        {
+            var connection = c.First();
+
+            return new ConnectionDto()
+            {
+                Party = Convert(connection.To),
```

Comment:
Samme her: basert på retting er det enten To eller From som skal er Party på responsen